src/app/api/tools/cowswap/orderFlow.ts

```diff
@@ -55,33 +66,26 @@ export async function orderRequestFlow({
     ...quoteResponse.quote,
     ...applySlippage(quoteResponse.quote, slippageBps),
   };
+
+  quoteResponse.quote.appData =
+    "0x5a8bb9f6dd0c7f1b4730d9c5a811c2dfe559e67ce9b5ed6965b05e59b8c86b80";
+  // TODO: This shit is too Slow.
+  // await buildAndPostAppData(
+  //   orderbook,
+  //   "bitte.ai/CowAgent",
+  //   referralAddress,
+  //   {
+  //     recipient: partnerAddress,
+  //     bps: partnerBps,
+  //   },
+  // );
```

Comment:
Reintroduce Dynamic AppData (without using that shitty [@cowprotocol/app-data](https://www.npmjs.com/package/@cowprotocol/app-data/v/1.1.0-RC.0) package). Note that much of the package functionality can be implemented synchronously.